src/Proto.OpenTelemetry/OpenTelemetryExtensions.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Diagnostics;
+
+namespace Proto.OpenTelemetry
+{
+    public delegate void ActivitySetup(Activity? activity, object message);
+
+    public static class OpenTelemetryExtensions
```

Comment:
How about extension that adds source to `TracerProviderBuilder`

```csharp
public static TracerProviderBuilder AddProtoActorInstrumentation(this TracerProviderBuilder builder) 
    => builder.AddSource(ProtoTags.ActivityName);
```

Reply:
Good idea, added